### PR TITLE
op-guide: add documentation to remove a TiKV

### PR DESCRIPTION
**Pull request by huachaohuang**

### CODE REVIEW

File: op-guide/binary-deployment.md [hunk 1]
```diff
@@ -171,8 +171,27 @@ cd tidb-latest-linux-amd64-centos6
 动态新加入一个新的 TiKV 服务是非常容易的，我们可以直接启动一个 TiKV 服务，PD 会自动检测到，
 并开始做整个集群的 balance，将其他 TiKV 的数据移动到新加入的 TiKV 里面。
 
-我们也能够显示的告诉 PD 去删除某个 TiKV。PD 会认为这个 TiKV 已经是 dead 了，如果某一个 region 
-原先有数据属于这个 TiKV，PD 就会选择一个新的 TIKV 给这个 region，重新做 balance。
+我们也能够显式的告诉 PD 去删除某个 TiKV。PD 会先把这个 TiKV 标记为正在下线的状态，
```
Comment (ngaut) [inline op-guide/binary-deployment.md]: s/TiKV/tikv-server/

Comment (siddontang) [inline op-guide/binary-deployment.md]: maybe `TiKV 服务` or `TiKV 实例`

Comment (huachaohuang) [inline op-guide/binary-deployment.md]: Seems very redundant, but I am fine with that if you insist.

**Comment by qiuyesuifeng** [comment]
LGTM

**Comment by ngaut** [comment]
LGTM

**Pull request merged**